server/celery_backend/tasks/upload_feedback_dump.py

```diff
@@ -0,0 +1,47 @@
+import csv
+import io
+from datetime import datetime
+
+from celery_backend.tasks.database import AppDatabase, LogDatastore
+from module.services.model.feedback import Feedback
+
+from ..celery_app import app
+from . import constants
+
+feedback_store = LogDatastore()
+
+
+@app.task(name="upload.feedback.dump")
+def upload_feedback_dump() -> None:
+    """Uploads feedback dumps to cloud storage"""
+    file = io.StringIO()
+    csv_writer = csv.writer(file)
+    app_db = AppDatabase()
+
+    collection = app_db.get_collection("feedback")
```

Comment:
Can we move this outside the task function?

Reply:
Doesn't matter lol, anyway it's run once a month

Reply:
Done it anyway lol